src/test.js

```diff
@@ -500,6 +509,13 @@ function synchronize( callback, priority ) {
 	}
 }
 
+function randomNumberGenerator( seed ) {
```

Comment:
Would that mean the only operation done in `src/core/config.js` is `config.random = urlParams.random`? We then do the actual seed generation at this point?